resources/templates/md/posts/2019-12-27.md

```diff
@@ -0,0 +1,25 @@
+{:title "ClojureTW 週報 2019-12-27"
+:layout :post
+:author "coldnew"
+:tags  ["clojure"]
+:toc true}
+
+# ClojureTW
+
+* 今年我們幾乎每週三都會在伯朗咖啡聚會。具體聚會的地點及詳細時間請關注 [meetup](https://www.meetup.com/Clojure-tw/events/) 的通知。
```

Comment:
That's no longer at the Coffee shop every week. It's also at Gandi every 2 weeks.